### ISSUE TITLE
.gitignore with any "!" prefix item causes eleventy to not process any files

### ISSUE THREAD

**Issue by bfelda** [11ty/eleventy#677]
**Describe the bug**
If your gitignore contains "!" prefixed items, such as the default VSCode items, eleventy will fail to process any files.  

.gitignore
```
# IDE - VSCode
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
.history/*
```

**To Reproduce**
1. Add the above to your .gitignore file
2. run eleventy
3. note "Processed 0 files in x seconds"
4. note _site dir is added but is empty

**Expected behavior**
files get processed by running eleventy

**Screenshots**

**Environment:**
 - windows 10
 - Eleventy Version 0.9.0

**Additional context**

**Comment by ogdenstudios** [comment]
Confirmed with the example `.gitignore` on OSX 10.14.6 and Eleventy 0.9.0. 

This doesn't seem to be an issue with Eleventy 0.8.3

**Comment by peter-sharp** [comment]
Confirmed on Windows 10 with the following .gitignore using Eleventy 0.9.0:
```
node_modules
.env
_site/*
!_site/assets
```

**Comment by Ryuno-Ki** [comment]
As workaround you can copy your `.gitignore` into an `.eleventyignore`, remove the lines starting with the exclamation mark and [tell eleventy to not look into the gitignore](https://www.11ty.io/docs/ignores/#opt-out-of-using-.gitignore).

**Comment by zygimantas** [comment]
I was also hit by this issue.
Additionally, I have noticed that with such config:
```
const input = "./src/Website/Areas/XXXXXAmp";

config.setUseGitIgnore(false);
config.addPassthroughCopy(`${input}/resources`);
config.addPassthroughCopy(`${input}/robots.txt`);
config.addPassthroughCopy(`${input}/favicon.ico`);
```

where `/Amp` directory exists and `/XXXXXAmp` does not, it fails, as expected in 0.8.3 with TemplatePassthroughManagerCopyError, but in 0.9.0, the output is:

`Copied 3 items and Processed 0 files in 0.02 seconds (v0.9.0)`

Of course, no files are copied, because `/XXXXXAmp` does not exist.

**Comment by philiprenich** [comment]
Using the `DEBUG` mode shows that Eleventy fails on the doubled `!`. I'm shown this string in output `!./!.vscode` where I had `!.vscode/` in the .gitignore

**Comment by zachleat** [comment]
Fixed by #709.
